components/infobox/wikis/starcraft2/infobox_league_custom.lua

```diff
@@ -0,0 +1,602 @@
+	if exchangeDate and currency and currency ~= 'USD' then
+		if localPrize then
+			local usdPrize = mw.ext.CurrencyExchange.currencyexchange(
+				localPrize,
+				currency,
+				'USD',
+				exchangeDate
+			)
+			if type(usdPrize) == 'number' then
+				return usdPrize
+			end
+		end
+	end
+
+	return nil
+end
+
+function CustomLeague:_displayPrizeValue(value, numDigits)
+	if value == 0 or value == '0' then
+		return '-'
+	end
+
+	numDigits = tonumber(numDigits or 0) or 0
+	local factor = 10^numDigits
+	value = math.floor(value * factor + 0.5) / factor
+
+	--split value into
+	--left = first digit
+	--num = all remaining digits before a possible '.'
+	--right = the '.' and all digits after it (unless they are all 0 or do not exist)
+	local left, num, right = string.match(value, '^([^%d]*%d)(%d*)(.-)$')
+	if right:len() > 0 then
+		local decimal = string.sub('0' .. right, 3)
+		right = '.' .. decimal .. string.rep('0', 2 - string.len(decimal))
+	end
+	return left .. (num:reverse():gsub('(%d%d%d)','%1,'):reverse()) .. right
+end
+
+function CustomLeague:_cleanPrizeValue(value, currency, oldHasPlus, oldHasText)
+	if String.isEmpty(value) then
+		return nil, nil, nil
+	end
+
+	--remove currency abbreviations
+	value = value:gsub('<abbr.*abbr>', ''):gsub(',', '')
+
+	--remove currency symbol
+	if currency then
+		Template.safeExpand(mw.getCurrentFrame(), 'Local currency', {currency:lower()})
+		local symbol = Variables.varDefineMulti('localcurrencysymbol', 'localcurrencysymbolafter') or ''
+		value = value:gsub(symbol, '')
+	else
+		value = value:gsub('%$', '')
+	end
+
+	--remove white spaces and &nbsp;
+	value = value:gsub('%s', ''):gsub('&nbsp;', '')
+
+	--check if it has a '+' at the end
+	local hasPlus = string.match(value, '%+$')
+	if hasPlus then
+		value = value:gsub('%+$', '')
+		hasPlus = true
+	end
+
+	--check if additional non numbers are present
+	local hasText = string.match(value, '[^%.%d]')
+	value = tonumber(value)
+
+	return value, hasText or oldHasText, hasPlus or oldHasPlus
+end
+
+function CustomLeague:addCustomContent(infobox, args)
+	--player breakdown
+	local playerRaceBreakDown = CustomLeague._playerRaceBreakDown(args) or {}
+	local playerBreakDownEvent = CustomLeague._playerBreakDownEvent(args) or {}
+	local playerNumber = playerRaceBreakDown.playerNumber or playerBreakDownEvent.playerNumber
+	infobox:header('Player Breakdown', playerNumber)
+	infobox:cell('Number of players', playerNumber)
+	infobox:fcell(CustomLeague._playerBreakDownDisplay(playerRaceBreakDown.display))
+	infobox:fcell(CustomLeague._playerBreakDownDisplay(playerBreakDownEvent.display))
+	Variables.varDefine('tournament_playerNumber', playerNumber)
+
+	--teams section
+	if args.team_number or (not String.isEmpty(args.team1)) then
+		Variables.varDefine('is_team_tournament', 1)
+		infobox:header('Teams', true)
+	end
+	infobox:cell('Number of teams', args.team_number)
+	if not String.isEmpty(args.team1) then
+		infobox:header('Teams', true)
```

Comment:
This can get you into a situation where you have two headers called `Teams`, right?